micrometer-core/src/main/java/io/micrometer/core/instrument/binder/cache/HazelcastCacheMetrics.java

```diff
@@ -45,7 +44,7 @@
      * @param <V>      The cache value type.
      * @return The instrumented cache, unchanged. The original cache is not wrapped or proxied in any way.
      */
-    public static <K, V, C extends IMap<K, V>> C monitor(MeterRegistry registry, C cache, String... tags) {
+    public static <K, V, C> C monitor(MeterRegistry registry, C cache, String... tags) {
```

Comment:
All those generics are useless now aren't they?

Reply:
Indeed they are (unfortunately). I've polished this in 8f1bff3a94dce5ca105b288ddf7def8da109f36d